tests/api_fastapi/conftest.py

```diff
@@ -36,3 +36,12 @@ def create_test_client(apps="all"):
         return TestClient(app)
 
     return create_test_client
+
+
+@pytest.fixture(scope="module")
+def dagbag():
+    from airflow.models import DagBag
+
+    dagbag_instance = DagBag(include_examples=True, read_dags_from_db=False)
+    dagbag_instance.sync_to_db()
+    return dagbag_instance
```

Comment:
If you put the fixture at the `conftest.py` level, do you mind also using it in other tests, and replace.

Maybe `test_dag` in `test_dag_sources.py` can reuse this fixture:
```
@pytest.fixture
def test_dag(dagbag):
    return dagbag.dags[TEST_DAG_ID]
```

Reply:
Yes this works, but if you don't mind, shall I pick this up in a separate PR? Have some other cleanup to do in tests so can do it together if that's okay.